README.md

```diff
@@ -43,6 +43,8 @@ git clone git@github.com:catalyst/moodle-tool_pdfpages.git <moodledir>/admin/too
 
 Use of the converter requires programmatic access, there in no frontend associated with this plugin, so you need to develop another module, or add this plugin to the dependencies of an existing Moodle plugin.
 
+> Only users with the system level capability `tool/pdfpages:createaccesskey` can conducted conversions, as this is required to create the single use access key for the headless browser session.
```

Comment:
how that would work if we would like to get users to generate pdf files by clicking an link in the browser? From the business point of view  pdfpages:createaccesskey does mean nothing. They shouldn't be aware about what's happening under the hood. I think we should rather call this cap something like "generate pdf from URL"  which will make more sense.

Reply:
Also it feels like we should check that permissions in index.php ?

Reply:
> how that would work if we would like to get users to generate pdf files by clicking an link in the browser?

This plugin API isn't designed for this use case at the moment, it is designed for the conversion to be offloaded to a scheduled and/or adhoc task and the link should serve the converted file once conversion is complete. So if live conversion was to occur, the user would either require the capability to generate PDFs, or conversion would need to be offloaded to another process (like an adhoc task) otherwise this would be an enhancement and there may be security concerns around allowing any user to click on a link and conduct conversion immediately.

> From the business point of view pdfpages:createaccesskey does mean nothing. They shouldn't be aware about what's happening under the hood. I think we should rather call this cap something like "generate pdf from URL" which will make more sense.

Agreed, amended capability to more accurately reflect the purpose.

Reply:
> Also it feels like we should check that permissions in index.php ?

Added a capability check to proxy following user login